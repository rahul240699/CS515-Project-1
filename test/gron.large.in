python3 prog/gron.py large.json