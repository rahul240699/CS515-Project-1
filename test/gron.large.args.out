o = {};
o.web-app = {};
o.web-app.servlet = [];
o.web-app.servlet[0] = {};
o.web-app.servlet[0].servlet-name = "cofaxCDS";
o.web-app.servlet[0].servlet-class = "org.cofax.cds.CDSServlet";
o.web-app.servlet[0].init-param = {};
o.web-app.servlet[0].init-param.configGlossary:installationAt = "Philadelphia, PA";
o.web-app.servlet[0].init-param.configGlossary:adminEmail = "ksm@pobox.com";
o.web-app.servlet[0].init-param.configGlossary:poweredBy = "Cofax";
o.web-app.servlet[0].init-param.configGlossary:poweredByIcon = "/images/cofax.gif";
o.web-app.servlet[0].init-param.configGlossary:staticPath = "/content/static";
o.web-app.servlet[0].init-param.templateProcessorClass = "org.cofax.WysiwygTemplate";
o.web-app.servlet[0].init-param.templateLoaderClass = "org.cofax.FilesTemplateLoader";
o.web-app.servlet[0].init-param.templatePath = "templates";
o.web-app.servlet[0].init-param.templateOverridePath = "";
o.web-app.servlet[0].init-param.defaultListTemplate = "listTemplate.htm";
o.web-app.servlet[0].init-param.defaultFileTemplate = "articleTemplate.htm";
o.web-app.servlet[0].init-param.useJSP = false;
o.web-app.servlet[0].init-param.jspListTemplate = "listTemplate.jsp";
o.web-app.servlet[0].init-param.jspFileTemplate = "articleTemplate.jsp";
o.web-app.servlet[0].init-param.cachePackageTagsTrack = 200;
o.web-app.servlet[0].init-param.cachePackageTagsStore = 200;
o.web-app.servlet[0].init-param.cachePackageTagsRefresh = 60;
o.web-app.servlet[0].init-param.cacheTemplatesTrack = 100;
o.web-app.servlet[0].init-param.cacheTemplatesStore = 50;
o.web-app.servlet[0].init-param.cacheTemplatesRefresh = 15;
o.web-app.servlet[0].init-param.cachePagesTrack = 200;
o.web-app.servlet[0].init-param.cachePagesStore = 100;
o.web-app.servlet[0].init-param.cachePagesRefresh = 10;
o.web-app.servlet[0].init-param.cachePagesDirtyRead = 10;
o.web-app.servlet[0].init-param.searchEngineListTemplate = "forSearchEnginesList.htm";
o.web-app.servlet[0].init-param.searchEngineFileTemplate = "forSearchEngines.htm";
o.web-app.servlet[0].init-param.searchEngineRobotsDb = "WEB-INF/robots.db";
o.web-app.servlet[0].init-param.useDataStore = true;
o.web-app.servlet[0].init-param.dataStoreClass = "org.cofax.SqlDataStore";
o.web-app.servlet[0].init-param.redirectionClass = "org.cofax.SqlRedirection";
o.web-app.servlet[0].init-param.dataStoreName = "cofax";
o.web-app.servlet[0].init-param.dataStoreDriver = "com.microsoft.jdbc.sqlserver.SQLServerDriver";
o.web-app.servlet[0].init-param.dataStoreUrl = "jdbc:microsoft:sqlserver://LOCALHOST:1433;DatabaseName=goon";
o.web-app.servlet[0].init-param.dataStoreUser = "sa";
o.web-app.servlet[0].init-param.dataStorePassword = "dataStoreTestQuery";
o.web-app.servlet[0].init-param.dataStoreTestQuery = "SET NOCOUNT ON;select test='test';";
o.web-app.servlet[0].init-param.dataStoreLogFile = "/usr/local/tomcat/logs/datastore.log";
o.web-app.servlet[0].init-param.dataStoreInitConns = 10;
o.web-app.servlet[0].init-param.dataStoreMaxConns = 100;
o.web-app.servlet[0].init-param.dataStoreConnUsageLimit = 100;
o.web-app.servlet[0].init-param.dataStoreLogLevel = "debug";
o.web-app.servlet[0].init-param.maxUrlLength = 500;
o.web-app.servlet[1] = {};
o.web-app.servlet[1].servlet-name = "cofaxEmail";
o.web-app.servlet[1].servlet-class = "org.cofax.cds.EmailServlet";
o.web-app.servlet[1].init-param = {};
o.web-app.servlet[1].init-param.mailHost = "mail1";
o.web-app.servlet[1].init-param.mailHostOverride = "mail2";
o.web-app.servlet[2] = {};
o.web-app.servlet[2].servlet-name = "cofaxAdmin";
o.web-app.servlet[2].servlet-class = "org.cofax.cds.AdminServlet";
o.web-app.servlet[3] = {};
o.web-app.servlet[3].servlet-name = "fileServlet";
o.web-app.servlet[3].servlet-class = "org.cofax.cds.FileServlet";
o.web-app.servlet[4] = {};
o.web-app.servlet[4].servlet-name = "cofaxTools";
o.web-app.servlet[4].servlet-class = "org.cofax.cms.CofaxToolsServlet";
o.web-app.servlet[4].init-param = {};
o.web-app.servlet[4].init-param.templatePath = "toolstemplates/";
o.web-app.servlet[4].init-param.log = 1;
o.web-app.servlet[4].init-param.logLocation = "/usr/local/tomcat/logs/CofaxTools.log";
o.web-app.servlet[4].init-param.logMaxSize = "";
o.web-app.servlet[4].init-param.dataLog = 1;
o.web-app.servlet[4].init-param.dataLogLocation = "/usr/local/tomcat/logs/dataLog.log";
o.web-app.servlet[4].init-param.dataLogMaxSize = "";
o.web-app.servlet[4].init-param.removePageCache = "/content/admin/remove?cache=pages&id=";
o.web-app.servlet[4].init-param.removeTemplateCache = "/content/admin/remove?cache=templates&id=";
o.web-app.servlet[4].init-param.fileTransferFolder = "/usr/local/tomcat/webapps/content/fileTransferFolder";
o.web-app.servlet[4].init-param.lookInContext = 1;
o.web-app.servlet[4].init-param.adminGroupID = 4;
o.web-app.servlet[4].init-param.betaServer = true;
o.web-app.servlet-mapping = {};
o.web-app.servlet-mapping.cofaxCDS = "/";
o.web-app.servlet-mapping.cofaxEmail = "/cofaxutil/aemail/*";
o.web-app.servlet-mapping.cofaxAdmin = "/admin/*";
o.web-app.servlet-mapping.fileServlet = "/static/*";
o.web-app.servlet-mapping.cofaxTools = "/tools/*";
o.web-app.taglib = {};
o.web-app.taglib.taglib-uri = "cofax.tld";
o.web-app.taglib.taglib-location = "/WEB-INF/tlds/cofax.tld";